cmd/kubeapps-apis/plugins/helm/packages/v1alpha1/server.go

```diff
@@ -111,18 +111,15 @@ func parsePluginConfig(pluginConfigPath string) VersionsInSummary {
 	if err != nil {
 		// return default value of VersionsInSummary
 		return VersionsInSummary{MajorVersionsInSummary,
-			MinorVersionsInSummary, PatchVersionsInSummary}
+			MinorVersionsInSummary, PatchVersionsInSummary}, nil
```

Comment:
Isn't this still swallowing the error (in this case, that the file could not be read)? We should be returning an error when we have one.

Reply:
Initially, I thought not to break any backward compatibility with users who have not provided values.yaml plugin config. As this plugin config is present in the config file, we can now handle it as an error. I have corrected this now.

Reply:
Just generally, no need to resolve conversations... usually the person who starts the conversation will resolve it if they want to, but they really help to regain context when switching back and forth between reviews.

> Initially, I thought not to break any backward compatibility with users who have not provided values.yaml plugin config.
> As this plugin config is present in the config file, we can now handle it as an error. I have corrected this now.

Great. Yes, the volume was always mounted, so the error would have only occurred if there was some failure at the OS level. We just shouldn't be swallowing errors at all. There's usually a better way.